src/main/java/org/folio/rest/delegate/AbstractDatabaseOutputDelegate.java

```diff
@@ -0,0 +1,32 @@
+package org.folio.rest.delegate;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import org.camunda.bpm.engine.delegate.DelegateExecution;
+import org.camunda.bpm.engine.delegate.Expression;
+import org.folio.rest.service.DatabaseConnectionService;
+import org.folio.rest.workflow.model.EmbeddedVariable;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public abstract class AbstractDatabaseOutputDelegate extends AbstractWorkflowDelegate implements Output {
+
+  Expression designation;
+
+  private Expression outputVariable;
```

Comment:
Should this have no access modifier like the designation?

Reply:
Nah, it is not used by the concrete class. The getter and setter for that property are defined by the interface and only used within a default interface method `setOutput`.